### PR TITLE
126 name choice 3 disabled if 2 blank

### PR DESCRIPTION
**Pull request by perkinss**
*Issue #:*
bcgov/entity#126

*Description of changes:*
Validate the second choice based on whether the third choice is filled or not.
Provide error message when invalid

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/test/unit/specs/RequestInfoHeader.spec.js [hunk 1]
```diff
@@ -983,23 +983,44 @@ describe('RequestInfoHeader.vue', () => {
...
     });
 
-    it.only('has the third name field and save disabled when the second is blank', () => {
-      console.log(vm.$el.querySelector('#namechoice-2-input'));
+    it('Editing the third name choice when the second is blank puts an error on the second field', () => {
       expect(vm.$el.querySelector('#namechoice-2-input').value).toEqual('');
-      expect(vm.$el.querySelector('#namechoice-3-input').disabled).toEqual(true);
+      expect(vm.$el.querySelector('#namechoice-3-input').value).toEqual('THREE');
+      expect(vm.$el.querySelector('#namechoice-2-input-error')).not.toBeNull();
+      expect(vm.$el.querySelector('#namechoice-2-input-error').innerHTML).toEqual('The second name choice is required before a third choice.');
```
Comment (perkinss) [inline client/test/unit/specs/RequestInfoHeader.spec.js]: I think this needs one more test to make sure if the third is empty there isn't still an error message on the second... somehting like that

**Comment by katiemcgoff** [comment]
Closing PR for now, this is not going into the sprint.
